testkit/package.mill

```diff
@@ -3,13 +3,18 @@ package build.testkit
 import mill._
 
 object `package` extends RootModule with build.MillPublishScalaModule {
-  def moduleDeps = Seq(build.main.eval, build.main.util, build.main)
-  def ivyDeps = Agg(build.Deps.TestDeps.utest)
+  def moduleDeps = Seq(build.bsp, build.main.eval, build.main.util, build.main)
+  def ivyDeps = Agg(
+    build.Deps.bsp4j,
+    build.Deps.TestDeps.utest
+  )
```

Comment:
Why got `testkit` new dependencies? I can't see how this is needed.

Reply:
Should be fixed now